### DISCUSSION TITLE
ExternalId use in service

### DISCUSSION THREAD

**Discussion by rivlinpereira** [Netflix/dispatch#2514] [General]
Hi,

What does ExternalId in services point to? For eg If I want to page a team in PD does ExternalId point to that team name? Reason why i am asking this is I cant get the commander to get paged when I create an incident. Any thoughts on this would be appreciated.

**Comment by mvilanova** [comment]
For PD, it's the _service_ id. See configuration doc [here](https://hawkins.gitbook.io/dispatch/administration-guide/incident/plugins/configuring-pagerduty#oncall-service-configuration).

**Reply by rivlinpereira** [reply]
thanks that worked
